vaccine_feed_ingest/cli.py

```diff
@@ -247,6 +248,16 @@ def cli():
     """Run vaccine-feed-ingest commands"""
     dotenv.load_dotenv()
 
+    sentry_path = os.environ.get("SENTRY_DSN")
+    if sentry_path:
```

Comment:
Can you add a ENABLE_SENTRY env bar too?

I like the ability to disable a service if needed without needing to remove the configuration for it

Reply:
It also means you can log higher if enable sentry is true, but the config is missing